### PR TITLE
Spectro Cloud Grafana Dashboards

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR adds the following new content and changes:
- New Spectro Cloud Grafana packs page
- Updates to the Prometheus Operator Pack page. The ability to use persistent storage is documented.
- Fix to an FRP packs page docs page.

## Review Changes

💻 [Preview URL](https://deploy-preview-1352--docs-spectrocloud.netlify.app/clusters/cluster-management/monitoring/deploy-monitor-stack)

🎫 [PEM-2124](https://spectrocloud.atlassian.net/browse/PEM-2124)
🎫 [DOC-796](https://spectrocloud.atlassian.net/browse/DOC-796)
🎫 [PPD-1282](https://spectrocloud.atlassian.net/browse/PPD-1282)

### CODE REVIEW

File: content/docs/06-integrations/00-grafana-spectrocloud-dashboards.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: 'Spectro Cloud Grafana Dashboards'
+metaTitle: 'Spectro Cloud Grafana Dashboards'
+metaDescription: 'Learn more about the Spectro Cloud Grafana dashboard and how to use them.'
```
Comment (ritawatson) [inline content/docs/06-integrations/00-grafana-spectrocloud-dashboards.md:4]: Is it called "Spectro Cloud Grafana Dashboard"? or is it "Spectro Grafana Dashboard"? (I'm thinking about consistency with other Spectro Dashboards, but this one is new to me.)

Should "them" be "it"?

Line 17 below has plural Dashboards, so maybe this is a dashboard that consists of several components?

File: content/docs/06-integrations/00-grafana-spectrocloud-dashboards.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+![A grafana dashboard view of the cluster metric displaying pack status](/clusters_cluster-management_grafana_spectro_metrics.png)
+
+
+# Versions Supported
+
+**1.0.X**
+
+# Prerequisites
+
+* A host cluster that has the [Prometheus Operator pack](/integrations/prometheus-operator) `v45.4.X` or greater installed. Check out the [Deploy Monitoring Stack](/clusters/cluster-management/monitoring/deploy-monitor-stack) for instructions on how to deploy a monitoring stack.
```
Comment (ritawatson) [inline content/docs/06-integrations/00-grafana-spectrocloud-dashboards.md:42]:
```suggestion
* A host cluster that has the [Prometheus Operator pack](/integrations/prometheus-operator) `v45.4.X` or greater installed. Check out [Deploy Monitoring Stack](/clusters/cluster-management/monitoring/deploy-monitor-stack) for instructions on how to deploy a monitoring stack.
```

File: content/docs/06-integrations/00-grafana-spectrocloud-dashboards.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+The Spectro Cloud Grafana Dashboards require no additional configuration and the pack is designed to work out-of-the-box. 
+
+You can learn how to add the Spectro Cloud Grafana Dashboards to your cluster by following the steps outlined in the [Enable Monitoring on Host Cluster](/clusters/cluster-management/monitoring/deploy-agent/).
+
+<br />
+
+<WarningBox>
+
+Pods without the defined attributes `request` and `limit` will display no metrics data in the Grafana out-of-the-box Kubernetes Pods dashboard.
```
Comment (ritawatson) [inline content/docs/06-integrations/00-grafana-spectrocloud-dashboards.md:58]:
```suggestion
Pods without the defined attributes `request` and `limit` will not display metrics data in the Grafana out-of-the-box Kubernetes Pods dashboard.
```

File: content/docs/06-integrations/00-grafana-spectrocloud-dashboards.md
Comment (ritawatson) [inline content/docs/06-integrations/00-grafana-spectrocloud-dashboards.md:1]: Maybe it's just me, but I'm not sure we should refer to dashboards as plural. When I look at the prometheus-cluster-metrics pack image, I see it as a single dashboard.

File: content/docs/06-integrations/00-prometheus-cluster-metrics.md [hunk 4]
```diff
@@ -12,32 +12,15 @@ logoUrl: 'https://registry.spectrocloud.com/v1/prometheus-operator/blobs/sha256:
...
 import WarningBox from 'shared/components/WarningBox';
 import InfoBox from 'shared/components/InfoBox';
-import PointsOfInterest from 'shared/components/common/PointOfInterest';
-import Tooltip from "shared/components/ui/Tooltip";
 
-# Prometheus Cluster Metrics
+# Overview
 
-The Prometheus Cluster Metrics is an addon pack that exposes internal cluster resource metrics. You can access the information exposed by the pack in Grafana by visiting the **Spectro Cloud / Spectro Clusters** dashboard. The following information is exposed by the Prometheus Cluster Metrics pack. <br /> <br />
+The Prometheus Cluster Metrics pack exposes Palette-specific metrics data from host clusters to Prometheus. You can use this data to learn more about the state of your clusters, resource utilization, and more. Use the [Spectro Cloud Grafana Dashboards](/integrations/grafana-spectrocloud-dashboards) pack to access the metric data through Grafana dashboards.
```
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-cluster-metrics.md:18]:
```suggestion
The Prometheus Cluster Metrics pack exposes Palette-specific host cluster metrics to Prometheus. You can use this data to learn about the state of your clusters, resource utilization, and more. Use the [Spectro Cloud Grafana Dashboards](/integrations/grafana-spectrocloud-dashboards) pack to access the metric data through Grafana dashboards.
```

**Review by ritawatson — APPROVED** [review]
LGTM with comments.

File: content/docs/04-clusters/06-cluster-management/01.9-monitoring/10-deploy-monitor-stack.md [hunk 5]
```diff
@@ -418,24 +474,24 @@ In this guide, the following domains are used to expose the monitoring stack:
 
   </InfoBox>
 
-36. Create a public certificate for each domain. If you are using a public cloud provider, use the native certificate manager service to generate a public certificate that you can attach to the load balancers with minimal friction. On-prem, use the default certificate manager used by your organization. 
+42. Create a public certificate for each domain. If you are using a public cloud provider, use the native certificate manager service to generate a public certificate that you can attach to the load balancers with minimal friction. On-prem, use the default certificate manager used by your organization.
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/01.9-monitoring/10-deploy-monitor-stack.md:477]:
```suggestion
42. Create a public certificate for each domain. If you are using a public cloud provider, use the native certificate manager service to generate a public certificate that you can attach to the load balancers with minimal overhead. On-prem, use the default certificate manager used by your organization. 
```

File: content/docs/06-integrations/00-grafana-spectrocloud-dashboards.md (hunk 1 above)
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-grafana-spectrocloud-dashboards.md:4]:
```suggestion
metaDescription: 'Learn more about the Spectro Cloud Grafana dashboard and how to use it.'
```

**Pull request merged**